docs/plugins/haproxy.md

```diff
@@ -0,0 +1,36 @@
+# `haproxy` plugin
```

Comment:
See apache http comment. I think we should have a **Format** section and show the input / output

observiq format

input
```
Sep 29 01:16:02 oiq-int-haproxy haproxy[338572]: {"frontend_type":"tcp","process_concurrent_connections":1,"frontend_concurrent_connections":1,"backend_concurrent_connections":0,"server_concurrent_connections":0,"backend_queue":0,"server_queue":0,"queue_wait_time":0,"server_wait_time":0,"response_time":0,"session_duration":49,"termination_state":"----","retries":0,"client_ip":"10.33.104.161","client_port":41132,"frontend_ip":"10.33.104.161","frontend_port":"8083","frontend_ssl_version":"TLSv1.3","frontend_ssl_ciphers":"TLS_AES_256_GCM_SHA384","frontend_name":"tcp-json","backend_name":"apache","server_name":"apache","bytes_read":11173,"bytes_uploaded":121}
```
output
```json
{
  "timestamp": "2021-09-29T01:16:02-04:00",
  "severity": 0,
  "labels": {
    "file_name": "in",
    "frontend_ssl_ciphers": "TLS_AES_256_GCM_SHA384",
    "frontend_ssl_version": "TLSv1.3",
    "log_type": "haproxy",
    "plugin_id": "haproxy"
  },
  "resource": {
    "backend_name": "apache",
    "frontend_ip": "10.33.104.161",
    "frontend_name": "tcp-json",
    "frontend_port": "8083",
    "frontend_type": "tcp",
    "host": "oiq-int-haproxy",
    "server_name": "apache"
  },
  "record": {
    "backend_concurrent_connections": 0,
    "backend_queue": 0,
    "bytes_read": 11173,
    "bytes_uploaded": 121,
    "client_ip": "10.33.104.161",
    "client_port": 41132,
    "frontend_concurrent_connections": 1,
    "pid": "338572",
    "process_concurrent_connections": 1,
    "process_name": "haproxy",
    "queue_wait_time": 0,
    "response_time": 0,
    "retries": 0,
    "server_concurrent_connections": 0,
    "server_queue": 0,
    "server_wait_time": 0,
    "session_duration": 49,
    "termination_state": "----"
  }
}
```

Reply:
Same as apache http comment with adding a future separate doc page.